repaygithub/cactus
# Cactus 550: accordion alignment

[CACTUS-550](https://repayonline.atlassian.net/browse/CACTUS-550)

Following [this workaround](https://github.com/philipwalton/flexbugs/issues/231), the problem seems to be (obviously IE11) that a `min-height` property breaks other properties like `align-items`. 

Test: 

- Open Accordion, Basic usage story on IE11 
- Make sure that the elements inside the menu are centered.
- Test it on mobile and ipad views.
